WordPress/Classes/ViewRelated/Gutenberg/Collapsable Header/CollapsableHeaderViewController.swift

```diff
@@ -218,10 +231,54 @@ class CollapsableHeaderViewController: UIViewController, NoResultsViewHost {
         /* This should be overriden in a child class in order to enable support. */
     }
 
+    // MARK: - Format Nav Bar
+    private var originalNavBarAppearance: (barStyle: UIBarStyle, barTintColor: UIColor?, isTranslucent: Bool, shadowImage: UIImage?)?
+    private func cacheNavBarApperance() {
+        originalNavBarAppearance = (navigationController?.navigationBar.barStyle ?? .default,
+                navigationController?.navigationBar.barTintColor,
+                navigationController?.navigationBar.isTranslucent ?? true,
+                navigationController?.navigationBar.shadowImage)
+    }
+
+    func formatNavigationController() {
+        if #available(iOS 13.0, *) {
+            let newAppearance = UINavigationBarAppearance()
+            newAppearance.configureWithTransparentBackground()
+            newAppearance.backgroundColor = .clear
+            newAppearance.shadowColor = .clear
+            newAppearance.shadowImage = UIImage()
+            navigationItem.standardAppearance = newAppearance
+            navigationItem.scrollEdgeAppearance = newAppearance
+            navigationItem.compactAppearance = newAppearance
+            setNeedsStatusBarAppearanceUpdate()
```

Comment:
Can this be after the conditional?

Also, I was thinking maybe the version conditional can benefit from a comment. Wdyt?

Reply:
That's a good suggestion. I added some inline documentation and included `@available(iOS, obsoleted: 13.0, message: "See description on 'formatNavigationController'")` This won't actually prevent you from calling the function in iOS 13+ yet but provides some more documentation on the intended use. How does that look to you?

Reply:
Awesome :+1: